javalib/src/main/scala/java/util/TreeSet.scala

```diff
@@ -0,0 +1,222 @@
+package java.util
+
+import java.lang.Comparable
+import scala.collection.mutable
+import scala.math.Ordering
+
+import scala.collection.JavaConversions._
+
+class TreeSet[E] protected (ordering: Ordering[_ >: E], _comparator: Comparator[_ >: E])
```

Comment:
You should not add a new constructor unless it is `private`. Just like with the methods.